bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/html_basic/internal/HeadRendererBridgeImpl.java

```diff
@@ -100,28 +80,40 @@ public void encodeChildren(FacesContext facesContext, UIComponent uiComponent) t
 		List<UIComponent> headComponentResources = uiViewRoot.getComponentResources(facesContext, "head");
 		List<UIComponent> styleSheetResources = new ArrayList<UIComponent>();
 		List<UIComponent> scriptResources = new ArrayList<UIComponent>();
+		List<UIComponent> otherHeadResources = new ArrayList<UIComponent>();
```

Comment:
@stiemannkj1 What type of resources other than script or stylesheet might we encounter?